src/Codebreaker.WinUI/Converters/GameStatusToVisibilityConverter.cs

```diff
@@ -7,36 +7,37 @@ public class GameStatusToVisibilityConverter : IValueConverter
 {
     public object Convert(object value, Type targetType, object? parameter, string? language)
     {
-        static Visibility GetStartVisibility(GameMode gameMode) => 
-            (gameMode is not GameMode.NotRunning) ? Visibility.Collapsed : Visibility.Visible;
-
-        static Visibility GetRunningVisibility(GameMode gameMode) => 
-            (gameMode is GameMode.NotRunning) ? Visibility.Collapsed : Visibility.Visible;
-
-        static Visibility GetCancelVisibility(GameMode gameMode) =>
-            (gameMode is GameMode.Started or GameMode.MoveSet) ? Visibility.Visible : Visibility.Collapsed;
+        if (value is not GameMode gameMode)
+            throw new InvalidOperationException("GameStatusToVisibilityConverter received an incorrect value type");
 
         string uiCategory = parameter?.ToString() ?? throw new InvalidOperationException("Pass a parameter to this converter");
 
-
-        if (value is GameMode gameMode)
+        return uiCategory switch
         {
-            return uiCategory switch
-            {
-                "Start" => GetStartVisibility(gameMode),
-                "Running" => GetRunningVisibility(gameMode),
-                "Cancelable" => GetCancelVisibility(gameMode),
-                _ => throw new InvalidOperationException("Invalid parameter value")
-            };
-        }
-        else
-        {
-            throw new InvalidOperationException("GameStatusToVisibilityConverter received an incorrect value type");
-        }
+            "Start" => GetStartVisibility(gameMode),
+            "Running" => GetRunningVisibility(gameMode),
+            "Cancelable" => GetCancelVisibility(gameMode),
+            "Lost" => GetLostVisibility(gameMode),
+            "Won" => GetWonVisibility(gameMode),
+            _ => throw new InvalidOperationException("Invalid parameter value")
+        };
     }
 
-    public object ConvertBack(object value, Type targetType, object parameter, string language)
-    {
+    public object ConvertBack(object value, Type targetType, object parameter, string language) =>
         throw new NotImplementedException();
-    }
+
+    private static Visibility GetStartVisibility(GameMode gameMode) =>
```

Comment:
these methods are only used within the Convert method? Change to local functions